.github/workflows/build-and-publish.yml

```diff
@@ -0,0 +1,37 @@
+name: Build and Publish
+
+on:
+  push:
+
+jobs:
+  build-and-publish:
+    name: Build and Publish
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Log in to Docker Hub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Extract metadata (tags, labels) for Docker
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          images: ${{ vars.DOCKER_ORG }}/${{ github.event.repository.name }}
+          tags: |
```

Comment:
I don't remember what this detail is about. It maybe possible to just omit the `tags` object entirely.

Reply:
I don't know what it does either.  It was copied from a different project.